### DISCUSSION TITLE
Browser using Form Builder issue

### DISCUSSION THREAD

**Discussion by iedex** [area17/twill#2458] [Q&A]
I think here is a problem:
https://github.com/area17/twill/blob/adc1237699dfd80035cb6f4184b78a0bcd299b44/src/Services/Forms/Fields/Browser.php#L148
When using only one module, the browser is acting like using multiple modules and the params([...]) are not sent.

**Comment by ifox** [comment]
Hi @iedex can you please clarify? How are you instantiating the field?

**Comment by iedex** [comment]
Hi @ifox 
This is an example:
```
Browser::make()->name('arts')->label('Articles')->modules(['articles'])->params(['special' => 1])->max(24)->required(),
```
Reading the code from the link in issue I realized that, that if statement will never be true. I think that if count($modules) === 1 then $modules[0] is set. I removed the "!" from  && ! isset($modules[0]) and after that it worked. (just for test)

Before that, the browser had the module selector in header, with the only module, and the params were not sent to the endpoint.

After the change, the module selector in browser window dissapeared and the params were sent.

I'm not sure if that "if" is the problem...

**Comment by ifox** [comment]
Thanks. We'll review. In the meantime, see if this example helps: https://github.com/area17/twill/blob/3.x/examples/basic-page-builder/app/Http/Controllers/Twill/MenuLinkController.php#L27
